src/Nether.Analytics.DataLake.Jobs/DAU.usql

```diff
@@ -0,0 +1,35 @@
+﻿//PURPOSE: A USQL job to compute the Daily Active Users (DAUs) of the game 
+// RECOMMENDED: run transactional eventdata daily
+
+
+//Declare input/output destinations as variables
```

Comment:
Parameters should be DECLARE EXTERNAL in order for us to replace them in runtime